packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -20,29 +20,27 @@ export const defaultOptions = {
           siteUrl
         }
       }
-      allMarkdownRemark(
-        limit: 1000,
+      
+      allSitePage(
+        limit: 1000000000,
```

Comment:
You've chosen an arbitrarily large number here. I find that in many cases, the developer should simply use `Infinity`.